README.rst

```diff
@@ -26,7 +26,16 @@ Run this command in your terminal:
 Pre-processing
 --------------
 
-TBD
+Prepare ``training_dataset`` and ``development_dataset`` as follows:
+
+.. code-block:: python
+
+    from msnc.dataset import Dataset
+
+    training_dataset = Dataset(examples)
+    development_dataset = Dataset(examples, training_dataset.x_to_index)
+
+Note that ``examples`` is an array which consists of ``{'index': index, 'Xs': Xs, 'y': y}``, where ``index`` (int), ``Xs`` (array of str), and ``y`` (int) represent an index, inputs, and output of an example, respectively.
```

Comment:
[Question]

Is `Xs` always `array of str`?

I think it could be `array of str` or `array of array of str`.
(https://github.com/jun-harashima/msnc/blob/master/msnc/dataset.py#L31)

Reply:
`Xs` is `array of array of str` 🙇 https://github.com/jun-harashima/msnc/pull/54

(https://github.com/jun-harashima/msnc/blob/master/msnc/dataset.py#L18)